src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -722,6 +722,19 @@ public void deleteUser(long id) {
         complete(submit(req("DELETE", tmpl("/users/{id}.json").set("id", id)), handleStatus()));
     }
 
+    public User suspendUser(User user, boolean action) {
```

Comment:
https://support.zendesk.com/hc/en-us/articles/203690976-Suspending-a-user

"To unsuspend a user, follow the steps above and select Unsuspend access."